circle.yml

```diff
@@ -0,0 +1,21 @@
+machine:
+  xcode:
+    version: 8.2
+
+checkout:
+  post:
+    - git submodule update --init --recursive
+
+dependencies:
+  pre:
+    - bundle install
+    - gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+    - Scripts/install_swiftlint.sh
```

Comment:
This PR is a bit behind on the PRs in the other repos. Will update it here when the rest is done.